x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -294,6 +298,34 @@ export type AnomalyAlertSeverityType = ValuesType<
   typeof ANOMALY_ALERT_SEVERITY_TYPES
 >['type'];
 
+export const ANOMALY_ALERT_DETECTORS = [
+  {
+    type: 'latency',
+    label: i18n.translate('xpack.apm.alerts.anomalyDetector.latencyLabel', {
+      defaultMessage: 'latency',
+    }),
+  },
+  {
+    type: 'throughput',
+    label: i18n.translate('xpack.apm.alerts.anomalyDetector.throughputLabel', {
+      defaultMessage: 'throughput',
+    }),
+  },
+  {
+    type: 'failed_transaction_rate',
+    label: i18n.translate(
+      'xpack.apm.alerts.anomalyDetector.failedTransactionRateLabel',
+      {
+        defaultMessage: 'failed transaction rate',
+      }
+    ),
+  },
+] as const;
+
+export type AnomalyAlertDetectorType = ValuesType<
+  typeof ANOMALY_ALERT_DETECTORS
+>['type'];
```

Comment:
Do we need this? Isn't this just the same as `ApmMlDetectorType[]`?

Reply:
As I put in previous comments, the strings are different